packages/preview/kanji-practice/0.1.0/README.md

```diff
@@ -0,0 +1,61 @@
+## こんにちは！
+
+This is a small typst package that creates charts with selected kanji for practicing. Look is based on [@jensechu/kanji](https://github.com/jensechu/kanji).
+
+#### Usage
+
+```typ
+#import "@local/kanji-practice:0.1.0": practice, set-font
+
+#set page(margin: (x: 5em, y: 2em))
+// Change fonts for info text
+#set text(font: ("Shantell Sans", "Kiwi Maru"))
+
+// Optional, change font for kanji preview in table
+#set-font(draw: "Kiwi Maru")
+// Or, also change fonts for stroke order
+#set-font(draw: "Kiwi Maru", strokes: "KanjiStrokeOrders")
+
+// Select kanjis. Spaces will be replaced with blank charts
+#practice(kanji: "分日一国人年大")
+```
+
+Each selected kanji will look like:
+
+![](docs/typst-screenshot.png)
```

Comment:
It would be nice to have an actual alternative text here, that describes the image with one or two sentences.